x-pack/plugins/snapshot_restore/server/routes/api/repositories.test.ts

```diff
@@ -290,8 +286,8 @@ describe('[Snapshot and Restore API Routes] Repositories', () => {
     });
 
     it(`doesn't return repository plugins that are not installed on all nodes`, async () => {
-      const dataNodePlugins = ['repository-s3', 'repository-azure'];
-      const masterNodePlugins = ['repository-azure'];
+      const dataNodePlugins = ['repository-hdfs'];
+      const masterNodePlugins = [''];
```

Comment:
Thanks a lot for updating the tests as well, @jrodewig! I think an empty array (as in "no plugins") might be a better choice here. 
 ```suggestion
      const masterNodePlugins = [];
```